iz_helpers/ui.py

```diff
@@ -249,16 +249,14 @@ def on_ui_tabs():
 """
                         )
 
-            with gr.Column(scale=1, variant="compact"):
-                output_video = gr.Video(label="Output").style(width=512, height=512)
-                (
-                    out_image,
-                    generation_info,
-                    html_info,
-                    html_log,
-                ) = create_output_panel(
-                    "infinite-zoom", shared.opts.outdir_img2img_samples
-                )
+                with gr.Column(scale=1, variant="compact"):
+                    output_video = gr.Video(label="Output").style(width=512, height=512)
+                    output_panel = create_output_panel("infinite-zoom", shared.opts.outdir_img2img_samples)
+                    
+                    out_image = output_panel.gallery
+                    generation_info = output_panel.generation_info
+                    html_info = output_panel.infotext
+                    html_log = output_panel.html_log
```

Comment:
for compatibility, maybe

```suggestion
                if isinstance(output_panel, tuple):
                    (
                        out_image,
                        generation_info,
                        html_info,
                        html_log,
                    ) = output_panel
                else:
                    out_image = output_panel.gallery
                    generation_info = output_panel.generation_info
                    html_info = output_panel.infotext
                    html_log = output_panel.html_log
```

Reply:
Can you please retry with a clean AUTOMATIC1111 webui install? I don't get any error, just the warning about gradio style method deprecation.

This PR just fixes the return type of panel creation, if you want to make it compatible with < 1.8.0 you can file another PR

Reply:
yes, it runs ok with the latest version, but the plugin also needs to be compatible with the previous version, isn't it?